### ISSUE TITLE
Accessibility issue with latest rollout of Claude Code on Windows app

### ISSUE THREAD

**Issue by samisin** [anthropics/claude-code#48464]
### Preflight Checklist

- [x] I have searched [existing requests](https://github.com/anthropics/claude-code/issues?q=is%3Aissue%20label%3Aenhancement) and this feature hasn't been requested yet
- [x] This is a single feature request (not multiple features)

### Problem Statement

I'm raising this as a concern as a person with neurological vision problems, and the most recent update of Claude Code on the Windows app, with the high contract themes, i.e. white text on black background, and vice-versa.

This makes it really hard for me to use the app in long sessions, as it causes headaches. This is the worst update Anthropic has made since I started transitioning from OpenAI to Anthropic with my projects, and I'm a paying Max user.

I always use dark mode in apps as it lessens the strain on my eyes, but this update is a clusterf*ck for a person with visual disabilities.

### Proposed Solution

Make it possible to adjust the contrast in the Claude Code tab. This should be an easy fix. Or add an additional dark and light theme, that is not high contrast.

### Alternative Solutions

There are none.

### Priority

Critical - Blocking my work

### Feature Category

Other
